src/components/views/auth/PasswordLogin.js

```diff
@@ -320,12 +324,16 @@ class PasswordLogin extends React.Component {
                 <form onSubmit={this.onSubmitForm}>
                     { loginType }
                     { loginField }
-                    <input className={pwFieldClass} ref={(e) => {this._passwordField = e;}} type="password"
+                    <Field
+                        className={pwFieldClass}
+                        id="mx_PasswordLogin_password"
+                        ref={(e) => {this._passwordField = e;}}
```

Comment:
tbh I'm a little surprised the linter isn't screaming about this.

also TIL that this is the recommended way to do refs now.

Reply:
Hmm, several things to say here... 

* I happen to just be moving existing `refs` code here, as I didn't want to change that much since it's not directly related (other auth modules still use older strings refs, for example). (I was worried you'd be sad I was mixing too much in one PR if I also clean up those sorts of code style things!)
* Is it the spacing that you'd expect linting errors for?
* Since string refs are deprecated, we may want to enable the [`react/no-string-refs` linting rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-string-refs.md) and clean then up